ipie/addons/adafqmc/qmc/adafqmc.py

```diff
@@ -0,0 +1,486 @@
+import numpy as np
+import torch
+from torch.utils.checkpoint import checkpoint
+from ipie.addons.adafqmc.hamiltonians.hamiltonian import ham_with_obs, rot_ham_with_orbs
+from ipie.addons.adafqmc.trial_wavefunction.sdtrial import SDTrial
+from ipie.addons.adafqmc.walkers.rhf_walkers import Walkers, initialize_walkers, reorthogonalize, sr
+from ipie.addons.adafqmc.propagation.propagator import Propagator
+import time
+
+
+class Dist_Params:
```

Comment:
no underscores in class names (DIstParams), what is Dist?

Reply:
Dist means distributed

Reply:
I would just call this ADAFQMCParams, or QMCParams, there's no name conflict, the QMC is the relevant part of the name not Dist :)

Reply:
OK I'll change the name